org.eclipse.scanning.example/src/org/eclipse/scanning/example/malcolm/EPICSv4ExampleModel.java

```diff
@@ -13,7 +13,7 @@
 
 import org.eclipse.scanning.api.device.models.MalcolmModel;
 
-public class ExampleMalcolmModel extends MalcolmModel {
+public class EPICSv4ExampleModel extends MalcolmModel {
 	
 	// TODO extend this model with additional fields
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)